packages/preview/hust-cse-report/0.1.0/typst.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "hust-cse-report"
```

Comment:
Since this is an unofficial template, you'll need to add some sort of non-descriptive prefix to the package name. That way multiple packages serving the same purpose can coexist without one having an unfair advantage. See our [naming rules](https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules).